gogs/gogs
# chore: update version of mermaid to 10.9.1

## Describe the pull request

Mermaid file was obtained from <https://cdn.jsdelivr.net/npm/mermaid@10.9.1/dist/mermaid.min.js>.  Version 10.9.1 seems to be the current released version.

When working with mermaid `sequenceDiagram` I received syntax error message.  Updating version of mermaid resolved the issue.

Link to the issue: n/a

## Checklist


## Test plan

1. Start gogs server.
2. Create markdown file in repository or edit existing.
3. Add the following mermaid code to markdown file:

```txt
sequenceDiagram

    participant zerohedge-crawler
    participant resource-article-tables
    participant article-watcher
    participant ai_article-table
    participant dav-queue
    participant bert-processor
    participant site

    zerohedge-crawler ->> resource-article-tables: create records

    article-watcher ->> resource-article-tables: finds unsync records
    article-watcher ->> ai_article-table: sync records
    article-watcher ->> dav-queue: send AI processing request
    article-watcher ->> dav-queue: check Archive for processed requests
    article-watcher ->> ai_article-table: update short column of AI article

    bert-processor ->> dav-queue: read request queue
    bert-processor ->> bert-processor: process request with AI
    bert-processor ->> dav-queue: write response content to archive queue
    bert-processor ->> dav-queue: write response status to status queue

    site ->> ai_article-table: read DB table and display to user
```

4. replace txt in the block above with mermaid.  Marker txt was used to preserve text display format.
5. save file and view it in gogs. It shall render diagram without error messages.
